### PR TITLE
Add Immich icon

### PR DESCRIPTION
**Pull request by zcraber**
immich

**GitHub Stars:**
[8.1k](https://github.com/immich-app/immich)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
Why did you pick the hex value?
Colour [found](https://immich.app/img/logo.png) on their website.

### CODE REVIEW

**Comment by LitoMore** [comment]
Hi @alextran1502, could you help to review this icon? We need approval from you since our icon library only supports monochrome.

**Comment by alextran1502** [comment]
@LitoMore Looks good to me

**Comment by LitoMore** [comment]
@alextran1502 Thank you!

File: _data/simple-icons.json [hunk 1]
```diff
@@ -5446,6 +5446,11 @@
             "hex": "00E7C3",
             "source": "https://github.com/immerjs/immer/blob/7a5382899bc8b0bf5e21972a1c7db63f53e1d697/website/static/img/immer-logo.svg"
         },
+        {
+            "title": "Immich",
+            "hex": "4250AF",
+            "source": "https://raw.githubusercontent.com/immich-app/immich/main/design/immich-logo.svg"
```
Comment (LitoMore) [inline _data/simple-icons.json:5452]:
```suggestion
            "source": "https://github.com/immich-app/immich/blob/bba4c4418279b7dc87e0f4a0b346a4e81057a631/design/immich-logo.svg"
```

**Pull request merged**